service/controller/v18patch1/version_bundle.go

```diff
@@ -8,33 +8,18 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "kubernetes",
-				Description: "Updated Kubernetes to 1.12.1. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md",
+				Component:   "cloudconfig",
+				Description: "Updated k8scloudconfig to 3.7.1.",
 				Kind:        versionbundle.KindChanged,
 			},
 			{
-				Component:   "calico",
-				Description: "Updated to 3.2.3. Also the manifest has proper resource limits to get QoS policy guaranteed.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "kubernetes",
-				Description: "Enabled admission plugins: DefaultTolerationSeconds, MutatingAdmissionWebhook, ValidatingAdmissionWebhook.",
+				Component:   "kube-proxy",
+				Description: "Now gets installed and upgraded before Calico.",
```

Comment:
I don't know how to conform this message with the past tense BS.

Reply:
:man_shrugging: